server/server.go

```diff
@@ -363,3 +371,20 @@ func (svr *Server) GetStoreStats(bt store.BackendType) (*store.Stats, error) {
 
 	return nil, fmt.Errorf("store not found")
 }
+
+type HandlerError struct {
+	Err  error
+	Meta *commitments.CommitmentMeta
+}
+
+func (e *HandlerError) Error() string {
+	return fmt.Sprintf("HandlerError: %s", e.Err.Error())
```

Comment:
Should we also print the meta info in here?

Reply:
yes, great idea, thanks for the suggestion!